src/components/ProgressCircle/ProgressCircle.tsx

```diff
@@ -0,0 +1,101 @@
+type ProgressCircleProps = {
+	percent: number;
+	value: number;
+	colorVariant: keyof typeof COLOR_VARIANTS;
+	transparentVariant: keyof typeof TRANSPARENT_VARIANTS;
+	label: string;
+};
+
+const TRANSPARENT_VARIANTS = {
+	1: {
+		opacity: "0.5",
+	},
+	2: {
+		opacity: "1",
+	},
+};
+
+const COLOR_VARIANTS = {
```

Comment:
Please just create 4 variants and add opacity for them if you need, but personally i would not mess up with opacity, i would rather use the same blue color or add another one, slightly different

Reply:
I also changed a little how variants are defined, i think we should not use object, just simple array should be enough in our case:
https://github.com/Zorkann/future-dashboard/blob/c6da9c91bab29c97c58362d3e7d939a5fd391cd9/src/components/DottedCircleBlock/DottedCircleBlock.tsx

Reply:
done